jquery.xblock.js

```diff
@@ -164,33 +164,62 @@
                 },
                 listenTo: function(name, callback) {
                     $this.dispatcher.bind(name, callback);
+                },
+                children: function(element) {
+                    return $(element).data('xblock-children');
+                },
+                childMap: function(block, childName) {
+                    var children = this.children(block);
+                    for (var i = 0; i < children.length; i++) {
+                        var child = children[i];
+                        if (child.name == childName) {
+                            return child;
+                        }
+                    }
                 }
             };
         },
 
-        jsInit: function(options, root) {
+        initializeXBlocks: function(options, root) {
+            // Find and initialize any XBlocks that are descendants of 'root', and their descendants.
             var $this = this;
+            return $(root).highestDescendants('.xblock:not(.xblock-initialized)').map(function(idx, blockDOM) {
```

Comment:
@bradenmacdonald I'm not sure if the root block itself got initialized this way. Can you please double-check it?